index.html

```diff
@@ -7,13 +7,103 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" type="text/css" href="css/grid.css">
     <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" type="text/css" href="css/main.css">
 
     <title>Responsive grid project</title>
 </head>
 
-<body>
+<body class="container">
+            <header class="row">
+                <h3>Project 1: A small bussiness </h3>
+                <nav class="col-12 row">
+                   <ul class="col-12 row">
+                       <li>ALLAN JOHNSON
+                           <h6>Pressonal Life Coach</h6>
+                       </li>
+                       <li>Home</li>
+                       <li>About</li>
+                       <li>Services</li>
+                       <li>Motivational Reading</li>
+                       <li>Blog</li>
+                       <li>Contacts</li>
+                       <li>log in</li>
+                   </ul>
+                </nav>
+            </header>
+            <main>
+            <section class="row hero-section">
+                    <div class="col-12 img-background">
+                        <img src="images/picText4.jpg" alt="heroPicture" width="100%">
+                        <div class="img-text row">
+                            <h6 class="col-12 h-center">AMBITON IS THE FIRST STEP TOWARDS</h6>
+                            <h1 class="col-12 h-center">SUCCESS</h1>
+                            
+                                <button class="col-1"><h6>BOOK NOW</h6></button>
```

Comment:
If you want to make the text inside the button larger, give the button a CSS class, and use the `font-size` rule. `<h1>`, `<h2>`, etc., are only for headings (titles).